Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT

```diff
@@ -1,3 +1,1154 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:baabcf0c9248f75c3f48fbf6640cefdb172a5558e47cc9bf72e847ef994df334
-size 16956
+[SUMMARY]

```

Comment:
Was it intended to have this file here? Apparently you included it by mistake

Reply:
No.  New commit created which I think has only `Makefile`.

Reply:
I reverted the accidentally changed file. Here is how you do it:
```
$ git checkout issue/120-daytime 
Switched to branch 'issue/120-daytime'
Your branch is up to date with 'origin/issue/120-daytime'.

$ git pull
Already up to date.

#######################################
# Restore the last version from `develop` branch
#######################################
$ git checkout develop -- Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT

$ git add Vignettes/fileio/txt.Shimadzu/UV-VIS-NIR.TXT

#######################################
# Display all the changes that this branch introduces
#######################################
$ git diff develop
diff --git a/Makefile b/Makefile
index 6d66ca1..cc120c0 100644
--- a/Makefile
+++ b/Makefile
@@ -53,7 +53,7 @@ roxygenize: DESCRIPTION hyperSpec/R/*.R
        @echo "Roxygenize"
        @Rscript --vanilla -e "library (methods, quietly=TRUE, verbose = FALSE); library (devtools, quietly=TRUE, verbose = FALSE); document ('hyperSpec')"
 
-DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -daystart -not -ctime 0 -name "DESCRIPTION") #only if not modified today
+DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -not -ctime 1 -name "DESCRIPTION") # only if not modified in the past 24 hrs
        @echo update DESCRIPTION
        @sed "s/\(^Version: .*-\)\(20\)\?[0-9][0-9][0-1][0-9][0-3][0-9]\(.*\)$$/\1$(DATE)\3/" hyperSpec/DESCRIPTION > .DESCRIPTION
        @sed "s/\(^Date: .*\)20[0-9][0-9]-[0-1][0-9]-[0-3][0-9]\(.*\)$$/\1`date +%F`\2/" .DESCRIPTION > hyperSpec/DESCRIPTION

$ git commit

$ git push 
```